### PR TITLE
Finished flicker image

### PR DESCRIPTION
**Pull request by 7ibakana**
What do u guys think? @sunnycake , @abdala2020, @jx4515ga

### CODE REVIEW

**Review by sunnycake — COMMENTED** [review]
@abdala2020 You're more familiar with this api from helping Dina. I think it's best for you to check and merge it. Thank you.

**Comment by 7ibakana** [comment]
I think I got it @abdala2020 .  Let me know.

**Comment by jx4515ga** [comment]
from the review yesterday I think it worked as we were expecting, in my opinion, it's good to go

**Comment by jx4515ga** [comment]
just a heads up for some reason there's a conflict that must be resolved

File: api/flickr.py [hunk 1]
```diff
@@ -1,11 +1,60 @@
...
+import os
 
-def get_image(search_term):
 
-    # talk to flickr
-    # download image 
-    # save, create Image to disk
-    return Image.open('api/broccoli.png')
+def getImage():
+    item = input("Enter the food item you would like an image for: ")
```
Comment (sunnycake) [inline api/flickr.py]: @abdala2020 @7ibakana  I think all the user prompt for user input should be from ui.py, what do you think? This flickr.py should only be making the call to the api.  

update: ignore first comment. I'm just saw that there's a more recent commit. Looking at that right now.

**Review by sunnycake — COMMENTED** [review]
Looks good to me.

**Pull request merged**